modules/oe_paragraphs_demo/config/install/core.entity_form_display.node.oe_demo_landing_page.default.yml

```diff
@@ -19,15 +19,21 @@ content:
     settings: {  }
     third_party_settings: {  }
   field_oe_demo_body:
-    type: entity_reference_paragraphs
```

Comment:
This change is not reflected on the update hook.

Reply:
I checked with the guys, for demo module we don't write upgrade path.